### PR TITLE
Fix disappearing tab when exiting the channel

### LINKED ISSUE #1302: Channel view content when navigating out of it
https://user-images.githubusercontent.com/12646744/131502397-c86c269c-de04-4d77-b39a-a02d23862d1f.mov

Take a look at the video, when I navigate to `Home`, the content of ChannelView changes to the content of Information tab for a split second before the view disappears. I guess it's because of the tabs implementation. We should improve this to avoid unnecessary changes on the screen

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1302

### CODE REVIEW

**Review by drillprop — APPROVED** [review]
Looks good to me 👍🏻

File: src/views/viewer/ChannelView/ChannelView.tsx [hunk 1]
```diff
@@ -271,6 +272,12 @@ export const ChannelView: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
```
Comment (drillprop) [inline src/views/viewer/ChannelView/ChannelView.tsx]: Just as a sidenote - this is not related to the issue, but I think we could remove this comment and add instead: 
```tsx
  const initialRender = useRef(true)
  // At mount set the tab from the search params
  useEffect(() => {
    if (initialRender.current) {
      const tabIndex = TABS.findIndex((t) => t === currentTabName)
      if (tabIndex === -1) setSearchParams({ 'tab': 'Videos' }, { replace: true })
      initialRender.current = false
    }
  })
```

Comment (rafalpawlow) [reply src/views/viewer/ChannelView/ChannelView.tsx]: This way `useEffect` will be fired on every render

Comment (drillprop) [reply src/views/viewer/ChannelView/ChannelView.tsx]: I don't think so, keep in mind that I used here `useRef`. It should run only on mount. This is very similar to https://stackoverflow.com/a/53180013

Comment (rafalpawlow) [reply src/views/viewer/ChannelView/ChannelView.tsx]: Yeah, you are right. I didn't look closely, I only saw that there was no array 😉

**Review by kdembler — APPROVED** [review]

**Pull request merged**